### PR TITLE
change affiliation field from mandatory to optional

### PR DESCRIPTION
**Pull request by SandySun2000**
https://github.com/IBM-Blockchain/JSDKIntegrationSample/issues/2

### CODE REVIEW

File: src/main/java/org/cr22rc/NetworkConfig.java [hunk 1]
```diff
@@ -171,7 +171,9 @@ String getName() {
         }
 
         String getMspid() {
```
Comment (cr22rc) [inline src/main/java/org/cr22rc/NetworkConfig.java:173]: I would be careful here and in other places as I don't think these are optional.  If they're not present probably should throw some type of exeception.

File: src/main/java/org/cr22rc/NetworkConfig.java [hunk 2]
```diff
@@ -292,15 +299,21 @@ public String getEventURL() {
         }
 
         public String getEnrollId() {
```
Comment (cr22rc) [inline src/main/java/org/cr22rc/NetworkConfig.java:296]: Again I think there would always be an id here

File: src/main/java/org/cr22rc/NetworkConfig.java [hunk 3]
```diff
@@ -292,15 +299,21 @@ public String getEventURL() {
...
-            return value.getString("enrollId");
+            if(value.containsKey("enrollId"))
+                return value.getString("enrollId");
+            return null;
         }
 
         public String getAffiliation() {
```
Comment (cr22rc) [inline src/main/java/org/cr22rc/NetworkConfig.java:302]: Here this seems appropriate

**Review by cr22rc — COMMENTED** [review]
Please look at comments.

File: src/main/java/org/cr22rc/NetworkConfig.java (hunk 1 above)
Comment (SandySun2000) [reply src/main/java/org/cr22rc/NetworkConfig.java:173]: If the field is required, should I remove the checking?

Comment (cr22rc) [reply src/main/java/org/cr22rc/NetworkConfig.java:173]: That's one option. I would think better throwing an exception with an error that the field was missing from the NetworkConfig document.

Comment (cr22rc) [reply src/main/java/org/cr22rc/NetworkConfig.java:173]: Have you done some testing with this change ?

Comment (SandySun2000) [reply src/main/java/org/cr22rc/NetworkConfig.java:173]: I only do the testing with "affiliation" change from the Jason file. I will leave those fields untouched then

File: README.md [hunk 4]
```diff
@@ -1,50 +1,56 @@
-# BMXHyperledgerFabricJSDKIntegrationSample
+This document shows how to integrate the IBM hyperledger Java SDK with IBM bluemix blockchain. To use Bluemix IBM Blockchain Service, follow https://github.com/IBM-Blockchain/marbles/blob/v4.0/docs/use_bluemix_hyperledger.md to create a Blockchain Network in IBM Bluemix, peers (up to 3) and a channel. 
 
 ## Hyperledger Fabric Java SDK  Version
- This code is dependent on a 1.1.0-SNAPSHOT
+ Requires Java SDK 1.1.0-SNAPSHOT
```
Comment (cr22rc) [inline README.md]: Probably should be Hyperledger Fabric Java SDK 1.1.0-SNAPSHOT

**Pull request merged**